<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>453</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>过敏原样本号替换小程序</string>
  </property>
  <widget class="QPushButton" name="pushViewData">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushDataDump">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxSample">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>样本信息</string>
   </property>
   <widget class="QTableView" name="tvSampleInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxResult">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>341</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>结果信息</string>
   </property>
   <widget class="QTextBrowser" name="textResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupSourceFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>721</width>
     <height>59</height>
    </rect>
   </property>
   <property name="title">
    <string>源数据文件</string>
   </property>
   <widget class="QPushButton" name="pushSourceFile">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textSourceFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushSampleInfo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>获取数据</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
